.moban.yaml

```diff
@@ -10,25 +10,29 @@ version: 0.1
 release: 0.1
 build_version: 0.1
 package_module: pyflakes_bears
-docs_output_dir: false
+docs_output_dir: docs
 ci_directory: .circleci
 
 dependencies:
   - coala==0.12.0.dev20180101025653
   - coala-bears==0.12.0.dev20171110210444
   - pyflakes~=1.5.0
 
+packages:
+  - pyflakes_bears
+  - pyflakes_generic_plugins
+
 configuration:
   template_dir:
-    - .moban.dt/
     - ../coala-mobans/templates/
     - ../coala-mobans/assets/
     - ../setupmobans/templates/
+    - .moban.dt/
```

Comment:
are you sure this change is necessary?

Reply:
No. Didn't had any effect. Would revert it.